app/grandchallenge/reader_studies/templates/reader_studies/readerstudy_detail.html

```diff
@@ -170,6 +178,11 @@ <h3>Admin Info</h3>
                         Each reader will read the cases in the same order.
                     {% endif %}
                 </p>
+                {% if object.questions_with_interactive_algorithm.exists and object.max_credits is not None %}
+                    <p>
+                        This reader study has {{ object.questions_with_interactive_algorithm.count|pluralize:"a question,questions" }} with an interactive algorithm. This doubles the credit consumption rate for this study.
```

Comment:
Better not to commit to how:

```suggestion
                        This reader study has {{ object.questions_with_interactive_algorithm.count|pluralize:"a question,questions" }} with an interactive algorithm. This increases the credit consumption rate for this study.
```